deploy/olm-catalog/che-workspace-crd/0.0.1/che-workspace-crd-operator.v0.0.1.clusterserviceversion.yaml

```diff
@@ -134,7 +134,7 @@ metadata:
         }
       ]
     capabilities: Basic Install
-  name: che-workspace-crd-operator.v0.0.1
+  name: che-workspace-controller.v0.0.1
```

Comment:
Does this file also need to be renamed?

Reply:
This file is renamed to `che-workspace-controller...`
`che-workspace-crd` olm package renamed to `che-workspace-operator`